aten/src/THC/THCGeneral.h.in

```diff
@@ -45,7 +45,7 @@
 #endif
 
 struct THCRNGState;  /* Random number generator state. */
-typedef struct THCStream THCStream;
+typedef struct CUDAStreamInternals THCStream;
```

Comment:
They're not really internals if we expose them like that 😄

Reply:
CUDAStreamImpl?

Reply:
It doesn't matter. It's just a bit funny